evaluation/views.py

```diff
@@ -17,6 +31,30 @@
 def evaluator(request):
     """Returns a random ayah for an expert to evaluate for any mistakes.
 
+    :param request: rest API request object.
+    :type request: Request
+    :return: Just another django mambo.
+    :rtype: HttpResponse
+    """
+    file_name = join(BASE_DIR, 'utils/data-uthmani.json')
+    with io.open(file_name, 'r', encoding='utf-8-sig') as file:
+        uthmani_quran = json.load(file)
+        file.close()
+
+    files = AnnotatedRecording.objects.filter(
+        file__gt='', file__isnull=False).order_by("?")[:7]
```

Comment:
Lets try to avoid `order_by("?")` early on? :smiley: 
Also, is there a need for `file__gt`?
```suggestion
        import random
        files = AnnotatedRecording.objects.filter(file__isnull=False)
        rand_files = random.sample(list(files), 7)
```

Reply:
Yes, I ran into issues when I did not include file__gt

Reply:
But agree about order_by("?")